templates/base.html

```diff
@@ -45,6 +45,22 @@
                         </a>
                     </li>
 
+                    {% if request.user.is_authenticated %}
+
+                    <li class="nav-item">
+                    <div class="btn-group">
+                        <button type="button" class="btn btn-success dropdown-toggle" data-bs-toggle="dropdown" aria-expanded="false">
+                            <i class="bi bi-person-circle"></i> {{request.user}}
+                        </button>
+                        <ul class="dropdown-menu">
+                          <li><a class="dropdown-item" href="{% url 'password_change' %}"><i class="bi bi-key me-1"></i>Change Password</a></li>
+                          <li><a class="dropdown-item" href="{% url 'logout' %}"><i class="bi bi-box-arrow-right me-1"></i>Sign out</a></li>
+                        </ul>
+                      </div>
+                    </li>
+
+                    {% else %}
+
```

Comment:
Is there a reason for removing the navbar as a separate file? I would prefer to have navbar and footer in separate files and `{% include %}` them. 
Also it is a bit superfluous to use `nav-item` if we're not using the bootstrap `navbar` class. 

_Navbars require a wrapping .navbar with .navbar-expand{-sm|-md|-lg|-xl} for responsive collapsing and color scheme classes._ See [link](https://getbootstrap.com/docs/4.0/components/navbar/)

Looks and works great though.

Reply:
Thanks for your comments!
The footer and navbar files that I removed were not ours. I now moved our own navbar and footer out.

Regarding the `nav-item`: that's under the [.nav](https://getbootstrap.com/docs/5.3/components/navs-tabs/#base-nav).
We might want to make the navigation collapsable and use `navbar` instead but I will leave this decision for later.